homeassistant/components/version/sensor.py

```diff
@@ -41,7 +41,7 @@ async def async_setup_entry(
 class VersionSensorEntity(VersionEntity, SensorEntity):
     """Version sensor entity class."""
 
-    _attr_icon = "mdi:package-up"
+    _attr_translation_key = "version"
```

Comment:
Would this not be better if moved to the entity description that is constructed on line 31?